d/docker-light-baseimage/cfssl_ppc64le.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/image/service-available/:ssl-tools/download.sh b/image/service-available/:ssl-tools/download.sh
+index a699f0f..5308768 100755
+--- a/image/service-available/:ssl-tools/download.sh
++++ b/image/service-available/:ssl-tools/download.sh
+@@ -1,34 +1,5 @@
+ #!/bin/bash -e
+ 
+-UARCH=$(uname -m)
+-echo "Architecture is ${UARCH}"
+-
+-case "${UARCH}" in
+-    
+-    "x86_64")
```

Comment:
Rather than removing all this code, can't we add a condition for ppc64le and build it from source if UARCH is ppc64le?

Reply:
Updated, please review